gensim/models/lda_worker.py

```diff
@@ -4,13 +4,13 @@
 # Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
-"""
-Worker ("slave") process used in computing distributed LDA. Run this script \
-on every node in your cluster. If you wish, you may even run it multiple times \
-on a single machine, to make better use of multiple cores (just beware that \
-memory footprint increases accordingly).
+"""Worker ("slave") process used in computing distributed LDA.
 
+Run this script on every node in your cluster. If you wish, you may even 
+run it multiple times on a single machine, to make better use of multiple
+cores (just beware that memory footprint increases accordingly).
```

Comment:
Please look at https://github.com/RaRe-Technologies/gensim/pull/1892, this is really good way how to document distributed stuff (instruction of running, showing arguments of script in automatic way, etc)